src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -193,7 +216,11 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
 
         widget.setFocused(true);
         widget.setName(this.customTagName);
-        widget.setWidgetContent(this.createContent());
+        if (this.state.content) {
```

Comment:
I prefer doing it the following way: `const widgetContent = this.state.content ? this.state.content : this.createContent();` and then `widget.setWidgetContent(widgetContent);`